api/app/src/main/kotlin/packit/controllers/LoginController.kt

```diff
@@ -50,6 +52,31 @@ class LoginController(
         return ResponseEntity.ok(token)
     }
 
+    @PostMapping("/login/jwt")
+    @ResponseBody
+    fun loginJWT(
+        @RequestBody @Validated user: LoginWithToken,
+    ): ResponseEntity<Map<String, String>>
+    {
+        if (jwtLoginService.audience == null) {
+            throw PackitException("jwtLoginDisabled", HttpStatus.FORBIDDEN)
+        }
+
+        val token = jwtLoginService.authenticateAndIssueToken(user)
+        return ResponseEntity.ok(token)
+    }
+
+    @GetMapping("/login/jwt/audience")
```

Comment:
That we can't change it (or at least not easily).

Most of the Packit API endpoints are only used by the Packit webapp, so they can easily be changed in a single PR. A few endpoints, mostly `/auth/login/api` and `/outpack` are used by orderly, and we couldn't change these easily. We'd have to change orderly, but also take into account the fact that people might have it installed already and worry about how that affect compatibility.

We need whatever returns the audience to be stable. It could be `/config`, or it could be a dedicated endpoint like I did here. Having a dedicated endpoint means we have to worry less about changing `/config`.